src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/WidgetThumbnail/index.spec.tsx

```diff
@@ -314,4 +339,52 @@ describe('<DashboardLiveWidgetThumbnail />', () => {
       refWidget: expect.any(HTMLDivElement),
     });
   });
+
+  it('fails to upload an image if the file is too large and displays an error toaster', async () => {
```

Comment:
I might need some help on this test. I've been working on it and supposed it would work as it does on the other resources (timedtext for example), but this mock seems not to send the expected metadata, which causes the test to fail because it cannot generate the right error message.